lesson_09/test_homework_09.py

```diff
@@ -0,0 +1,129 @@
+
+#Задача 1
+
+import unittest
+from homework_09 import reverse_string
+
+
+class TestReverseStringFunction(unittest.TestCase):
+
+    def test_reverse_string_positive(self):
+        input_string = "Привіт Світ"
+        expected_result = "тІвірпП"
+        result = reverse_string(input_string)
+        self.assertEqual(result, expected_result, "The string reversal is incorrect for positive test case.")
+
+    def test_reverse_string_negative(self):
+        input_string = ""  # Порожній рядок
+        expected_result = ""  # Порожній рядок залишається порожнім після реверсу
+        result = reverse_string(input_string)
+        self.assertEqual(result, expected_result, "The string reversal is incorrect for negative test case.")
+
+
+if __name__ == "__main__":
```

Comment:
It is no need to create __name__ == __main__ for each class. It is enough to create at least 1 for whole file.

Reply:
Thank you for your comments; I have fixed everything.